playwright-e2e/pages/dsm/participants/participants-page.ts

```diff
@@ -51,4 +51,14 @@ export default class ParticipantsPage {
     await input.toLocator().type(searchString);
     await Promise.all([this.page.locator('.fa-spinner').waitFor({ state: 'visible' }), this.searchButton().click()]);
   }
+
+  async filterMedicalRecordPts(): Promise<void> {
+    await this.page.locator("text=Customize View >> button").click();
+    await this.page.locator('text=Medical Record Columns').click();
+    await this.page.locator('text=Initial MR Received').check();
+    await this.page.locator("text=Search >> button").click();
+    await this.page.locator("text=Initial MR Received mm/dd/yyyy * Today >> button >> nth=2").click();
```

Comment:
Be aware, `nth=2` makes locator brittle and flaky.

Reply:
Yes, this is a very flakly locator. i could not think of a better way to activate this filter unfortunately. Most other options I tried could not locate the proper button. Do you have any suggestions for a better locator @aweng98 ?